src/main/java/uk/gov/hmcts/reform/hmc/config/MessageProcessor.java

```diff
@@ -44,26 +69,20 @@ public void processMessage(JsonNode message, MessageType messageType, Map<String
                 case AMEND_HEARING:
                     log.info("Message of type AMEND_HEARING received");
                     defaultFutureHearingRepository.amendHearingRequest(
-                        message,
-                        applicationProperties.get("caseListingID").toString()
+                        message, caseListingID
                     );
                     break;
                 case DELETE_HEARING:
                     log.info("Message of type DELETE_HEARING received");
                     defaultFutureHearingRepository.deleteHearingRequest(
-                        message,
-                        applicationProperties.get("caseListingID").toString()
+                        message, caseListingID
                     );
                     break;
                 default:
-                    log.info("Message has unsupported value for message_type");
-                    // add to dead letter queue - unsupported message type
-                    break;
+                    //should not get here as no more MessageType constants
```

Comment:
Can we still throw an exception here for robustness, the same as L46 please? In future if we introduce a new value to the enum for any reason but not the `switch` then we would hit the `default`.